dxfuse.go

```diff
@@ -445,8 +445,42 @@ func (fsys *Filesys) GetInodeAttributes(ctx context.Context, op *fuseops.GetInod
 	return nil
 }
 
+func (fsys *Filesys) removeInodeFile(ctx context.Context, oph *OpHandle, file File) error {
```

Comment:
```suggestion
func (fsys *Filesys) removeRemoteFile(ctx context.Context, oph *OpHandle, file File) error {
```

Since you have nicely refactored this into its own function, can you also invoke this function in `Unlink()`? To avoid two similar implementations for remote file removall.